addons/project/views/project_views.xml

```diff
@@ -95,6 +95,23 @@
             </field>
         </record>
 
+        <record id="project_task_action_sub_task" model="ir.actions.act_window">
+            <field name="name">Sub-tasks</field>
+            <field name="res_model">project.task</field>
+            <field name="view_mode">tree,kanban,form,calendar,pivot,graph,activity</field>
+            <field name="search_view_id" ref="project.view_task_search_form"/>
+            <field name="domain">[('id', 'child_of', active_id), ('id', '!=', active_id)]</field>
+            <field name="context">{'show_project_update': False}</field>
```

Comment:
Shouldn't we set the default_parent_id on the action? I have seen it is set on the stat button but I would have done it on the action in case it is used somewhere else. This would be consistent with the domain of the action I think